[2024-05-28T20:34:13.117+0530] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-05-28T20:34:13.171+0530] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: my_data_pipeline_dag.collect_data_1 manual__2024-05-28T15:04:07.228566+00:00 [queued]>
[2024-05-28T20:34:13.180+0530] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: my_data_pipeline_dag.collect_data_1 manual__2024-05-28T15:04:07.228566+00:00 [queued]>
[2024-05-28T20:34:13.180+0530] {taskinstance.py:2303} INFO - Starting attempt 1 of 2
[2024-05-28T20:34:13.207+0530] {taskinstance.py:2327} INFO - Executing <Task(BashOperator): collect_data_1> on 2024-05-28 15:04:07.228566+00:00
[2024-05-28T20:34:13.216+0530] {standard_task_runner.py:63} INFO - Started process 234773 to run task
[2024-05-28T20:34:13.265+0530] {standard_task_runner.py:90} INFO - Running: ['airflow', 'tasks', 'run', 'my_data_pipeline_dag', 'collect_data_1', 'manual__2024-05-28T15:04:07.228566+00:00', '--job-id', '39', '--raw', '--subdir', 'DAGS_FOLDER/my_first_dag.py', '--cfg-path', '/tmp/tmpvpbgea_d']
[2024-05-28T20:34:13.266+0530] {standard_task_runner.py:91} INFO - Job 39: Subtask collect_data_1
[2024-05-28T20:34:13.459+0530] {task_command.py:426} INFO - Running <TaskInstance: my_data_pipeline_dag.collect_data_1 manual__2024-05-28T15:04:07.228566+00:00 [running]> on host DESKTOP-J05K55U.
[2024-05-28T20:34:13.544+0530] {taskinstance.py:2644} INFO - Exporting env vars: AIRFLOW_CTX_DAG_EMAIL='airflow@example.com' AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='my_data_pipeline_dag' AIRFLOW_CTX_TASK_ID='collect_data_1' AIRFLOW_CTX_EXECUTION_DATE='2024-05-28T15:04:07.228566+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-05-28T15:04:07.228566+00:00'
[2024-05-28T20:34:13.545+0530] {taskinstance.py:430} INFO - ::endgroup::
[2024-05-28T20:34:13.546+0530] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2024-05-28T20:34:13.548+0530] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'Collect data/covid_19_europe.py']
[2024-05-28T20:34:13.562+0530] {subprocess.py:86} INFO - Output:
[2024-05-28T20:34:13.720+0530] {subprocess.py:93} INFO - /usr/bin/bash: line 1: Collect: command not found
[2024-05-28T20:34:13.720+0530] {subprocess.py:97} INFO - Command exited with return code 127
[2024-05-28T20:34:13.721+0530] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-05-28T20:34:13.727+0530] {taskinstance.py:2890} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/wsl/airflow_venv/lib/python3.10/site-packages/airflow/models/taskinstance.py", line 465, in _execute_task
    result = _execute_callable(context=context, **execute_callable_kwargs)
  File "/home/wsl/airflow_venv/lib/python3.10/site-packages/airflow/models/taskinstance.py", line 432, in _execute_callable
    return execute_callable(context=context, **execute_callable_kwargs)
  File "/home/wsl/airflow_venv/lib/python3.10/site-packages/airflow/models/baseoperator.py", line 400, in wrapper
    return func(self, *args, **kwargs)
  File "/home/wsl/airflow_venv/lib/python3.10/site-packages/airflow/operators/bash.py", line 243, in execute
    raise AirflowException(
airflow.exceptions.AirflowException: Bash command failed. The command returned a non-zero exit code 127.
[2024-05-28T20:34:13.735+0530] {taskinstance.py:1205} INFO - Marking task as UP_FOR_RETRY. dag_id=my_data_pipeline_dag, task_id=collect_data_1, execution_date=20240528T150407, start_date=20240528T150413, end_date=20240528T150413
[2024-05-28T20:34:13.751+0530] {standard_task_runner.py:110} ERROR - Failed to execute job 39 for task collect_data_1 (Bash command failed. The command returned a non-zero exit code 127.; 234773)
[2024-05-28T20:34:13.794+0530] {local_task_job_runner.py:240} INFO - Task exited with return code 1
[2024-05-28T20:34:13.822+0530] {taskinstance.py:3482} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-05-28T20:34:13.824+0530] {local_task_job_runner.py:222} INFO - ::endgroup::
